bowling.rb

```diff
@@ -0,0 +1,35 @@
+score = ARGV[0] #スクリプト実行時に渡された引数を参照する
+scores = score.split(",") #点数を分割する
+shots = [] #メソッドの呼び出しを定義
+scores.each do |t|
+  if t == "X"
+    shots << 10
+    shots << 0
+  else
+    shots << t.to_i
+  end
+end
+
+frames = shots.each_slice(2).to_a do |t| # 配列内の要素を2つずつ渡す
+  frames << t
+end
```

Comment:
ここは

```suggestion
frames = shots.each_slice(2).to_a
```

でよさそうですね。
僕は `to_a` にブロックをわたしたことがなかったので、リファレンスマニュアルで調べてみたのですが、 
https://docs.ruby-lang.org/ja/latest/method/Enumerable/i/entries.html
`each の呼び出し時に引数として渡されます。` なんですね。
なので、eachと同じように処理しているのかなと思いますが、 `to_a` は単純に配列化する( `to_i` とかと一緒で `transform to array` みたいなイメージです)ものなので、その返り値を使ってあげれば十分です :+1:

Reply:
上記、試しに削除してみたのですが、普通にプログラムが動いて驚きました...
ご確認ありがとうございます。